core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -829,6 +881,7 @@ class ZKConfig(props: VerifiableProperties) {
 
 object ZkPath {
   @volatile private var isNamespacePresent: Boolean = false
+  import scala.collection.JavaConversions._
```

Comment:
Perhaps it's simpler to just pass in the java version of the acl list and get rid of the implicit conversion.